boards/v2/qemu/qemu_cortex_a53/Kconfig.qemu_cortex_a53

```diff
@@ -2,13 +2,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_QEMU_CORTEX_A53
-	bool "Cortex-A53 Emulation (QEMU)"
-	depends on SOC_QEMU_CORTEX_A53
+	select SOC_QEMU_CORTEX_A53
 	select ARM64
 	select QEMU_TARGET
 
 config QEMU_CORTEX_A53_ETH_E1000
```

Comment:
this definition should go to `Kconfig` not `Kconfig.<board>`.

Please also check that `QEMU_CORTEX_A53_ETH_E1000` is still the right setting name.

Reply:
Fixed